it/testData/FinancialData1811.scala

```diff
@@ -70,7 +70,7 @@ object FinancialData1811 {
             "interestAccruingAmount" -> 12.10
           ),
           "documentPenaltyTotals" -> Json.arr(Json.obj(
-            "penaltyType" -> "LPP1",
+            "penaltyType" -> "LPP2",
```

Comment:
Any reason why this was changed to LPP2? It might be a bit picky but I think it's confusing to change this to accruing LPP2 given we know from the example data that it won't come back as a single item in an array.

You could also remove `postedChargeReference` field from here as it won't come back for accruing penalties and we aren't reading it anyway.